openhtf/conf.py

```diff
@@ -194,74 +196,25 @@ def __init__(self, flags, logger, lock, _functools, **kwargs):
     Args:
       logger: Logger to use for logging messages within this class.
       lock: Threading.Lock to use for locking access to config values.
-      _functools: Reference to the functools module so we can use it internally
-          for decorating methods.
       **kwargs: Modules we need to access within this class.
     """
-    self._flags = flags
     self._logger = logger
     self._lock = lock
-    self._functools = _functools
     self._modules = kwargs
     self._declarations = {}
+    self.ARG_PARSER = parser
```

Comment:
no, because the config module gets replaced at the end of the module, this has to be accessible on the Configuration object. I opted to keep the name as-is